docs/reference/migration/migrate_8_0/reindex.asciidoc

```diff
@@ -7,4 +7,9 @@ re-encode them when generating the search request for the remote host. This
 leniency has been removed such that all index-names are correctly encoded when
 reindex generates remote search requests.
 
-Instead, please specify the index-name without any encoding.
\ No newline at end of file
+Instead, please specify the index-name without any encoding.
+
+[float]
+==== Removal of types
+
+The `/{index}/{type}/_delete_by_query` and `/{index}/{type}/_update_by_query` REST endpoints have been removed in favour of `/{index}/_delete_by_query` and `/{index}/_update_by_query`. Since indexes can now only contain a single unnamed type these typed endpoints are obselete.
```

Comment:
I wonder if the phrasing 'indexes can now only contain a single unnamed type' could be confusing. We now provide 'typeless' index creation APIs, which to users seem like they don't introduce a type at all. Perhaps we could just say 'Since indexes no longer contain types, these typed endpoints are obsolete.'

Also small typo: obselete -> obsolete.

Reply:
++ good catch, I think your wording is much better. I'll change it in this and the other PR